Source/Mesh/InterfaceElementMethods.f90

```diff
@@ -39,6 +39,36 @@
 !      Created: May 16, 2013 11:17 AM 
 !      By: David Kopriva  
 !
+!Interface boundary steps
```

Comment:
We are actually not using that algorithm, but one that you and I worked out one day at my desk. They are different, and theirs is only cleanup free because it does the work at time of construction. Not saying anything bad about their algorithm, it's just not what's in the code here. It might be considered by someone in the future.

Reply:
Ah, okay. I just wanted to be sure to give proper credit.